### DISCUSSION TITLE
Docs: Tailwind scaffolding in Next.js Projects for separation of concerns

### DISCUSSION THREAD

**Discussion by Amansg4** [vercel/next.js#33482] [Help] [answered]
### What is the improvement or update you wish to see?

I would like to see a template on how we should organize scaffolding of CSS files when using tailwind. 

### Is there any context that might help us understand?

I know we can use global.css file for global variables, but what is the proper way of organizing CSS files for the sake of separation of concerns, route folders, and/or general organization? 

Are there any gotchas/caveats to separating CSS files?

Checked the with-tailwind example, and unfortunately it only provided the simplest example with global.css file(initial setup) and inline modifications.

And thanks! It really has been great getting to know Next.js btw. Still got loads to learn but having fun.

### Does the docs page already exist? Please link to it.

https://github.com/vercel/next.js/tree/canary/examples/with-tailwindcss

**Comment by balazsorban44** [comment, marked as answer]
I believe organizing files is a personal matter and hard to find a perfect structure. The few folders Next.js enforces are:

`public` for static assets: https://nextjs.org/docs/basic-features/static-file-serving
`pages` for file-based routing: https://nextjs.org/docs/basic-features/pages
`pages/api` for API routes: https://nextjs.org/docs/api-routes/introduction

Anything else is really up to the developer.
